rholang/src/main/scala/coop/rchain/rholang/interpreter/util/RevAddress.scala

```diff
@@ -0,0 +1,18 @@
+package coop.rchain.rholang.interpreter.util
+import coop.rchain.crypto.codec.Base16
+
+final case class RevAddress(val keyHash : Array[Byte])
+
+object RevAddress {
+  private val coinId = "000000"
+  private val version = "00"
+  private val prefix = Base16.decode(coinId + version)
+
+  private val tools = new AddressTools(prefix, keyLength = 32, checksumLength = 4)
```

Comment:
I think the keyLength should be 2048 at least. We also probably shouldn't restrict it to one possible value - rather allow 2048, and the subsequent powers of 2. Let's just have 2048 for now though.

Notice this doesn't affect the address length.

Reply:
I meant 2048 bits = 256 bytes, vs 256 bits = 32 bytes we have now.
But a quick google search suggests that Bitcoins uses 32-byte public keys?
I'm pretty sure we discussed 2048-bit keys. @cboscolo should we go full-bitcoin?

Reply:
I had the impression we are using Ed25519 which has a public key length of 256bits

Reply:
The RevAddress is the output of blake2b256(Ed25519_pub_key)

Reply:
@odeac I've seen a similar identifier string, but am unsure. Can you point to a specific place in code?

Reply:
I've missed @cboscolo's comment. I guess your combined input closes the case, Gentlemen :)